tools/deb/description-discovery

```diff
@@ -0,0 +1 @@
+Istio Pilot Discovery provides services discovery for the http://istio.io project.
```

Comment:
You mean "Istio Pilot server provides ... " ? Not sure how much text / boilerplate we should put there. Will add CRLF at the end.